user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIPage.md

```diff
@@ -1,12 +1,20 @@
 ---
-uid: GQIPage
+uid: GQI_GQIPage
 ---
 
-# GQIPage
+# GQIPage Class
 
-The *GQIPage* object has the following properties:
+## Definition
 
-| Property    | Type     | Required | Description                                                                            |
-| ----------- | -------- | -------- | -------------------------------------------------------------------------------------- |
-| Rows        | GQIRow[] | Yes      | The rows of the page.                                                                  |
-| HasNextPage | Boolean  | No       | True if additional pages can be retrieved, False if the current page is the last page. |
\ No newline at end of file
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+Represents a single page of rows.
+Every query result consists of one or more such pages.
+
+## Properties
+
+| Property | Type | Required | Description |
```

Comment:
Not sure if we need to keep this Required column.
But that's probably out of scope of this change.